hr_expense_sheet_payment_state/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "HR Expense Report Payment Status",
+    "summary": "Backport of the Payment Status (payment_state field) "
+    "in Expense Report which is introduced in Odoo Community Version 15.0",
+    "version": "14.0.1.0.0",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
```

Comment:
If this is an Odoo CE backport, shouldn't the author (and copyright) be Odoo SA?

Reply:
I'm not an expert, so if you definitely know pls suggest the right solution.
@yvaucher can you pls help defining correct license reference?

Reply:
@yankinmax I think you should add Odoo SA. to author and commit with odoo SA.
you can see example with https://github.com/OCA/payroll/pull/5

Reply:
@yankinmax Please have a look at the contributing rules on this section: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#10backporting-odoo-modules

Then, as it is a backport of a feature and not a full module, I'd suggest you do the following.
1. Copy the code from Odoo that you will have to work on, and you can commit this with odoo SA, or the commit's author.
2. Apply your changes on top of it in a second commit that you author yourself.
3. Add requirements from the contributing guidelines.